server/webhook.go

```diff
@@ -296,17 +306,21 @@ func (p *parsedJIRAWebhook) fromChangeLog(issue string) (string, string) {
 			p.event(eventUpdatedResolved)
 			return fmt.Sprintf("resolved %v", issue), ""
 
-		case item.Field == "status" && to == "Backlog":
+		case item.Field == "status" && lto == "in progress":
```

Comment:
@levb Instead of hardcoding to `in progress`, etc., I'm wondering if we're able to use the statuses defined in Jira.